tpls/mdspan/include/experimental/__p2630_bits/submdspan_mapping.hpp

```diff
@@ -168,7 +168,7 @@ struct deduce_layout_left_submapping<
   }
 
   MDSPAN_INLINE_FUNCTION
-  constexpr static bool layout_left_padded_value() {
```

Comment:
This is a member function so I'm surprised this caused problems.

Reply:
Hi Wolfgang.  Did you mean to commit that change?
I agree with Daniel's assessment.

Reply:
The CI is also complaining about calling a non-static member function in https://github.com/kokkos/kokkos/actions/runs/14995118601/job/42127392081?pr=8071.

Reply:
Yes, this was clearly a mistake -- thank you for fixing it, @masterleinad!